docs/use-mainnet/fund.mdx

```diff
@@ -94,4 +105,60 @@ Not all available tokens will drip directly onto Linea. If they are, you will se
 
 :::
 
-Specific tokens require specific bridges. If you want to bridge from Goerli to Linea, you can find the tokens, contract addresses, and associated bridges [here](./info-contracts.mdx#token-contract-addresses-and-bridges).
\ No newline at end of file
+Specific tokens require specific bridges. If you want to bridge from Goerli to Linea, you can find the tokens, contract addresses, and associated bridges [here](./info-contracts.mdx#token-contract-addresses-and-bridges).
+  </TabItem>
+  
+  <TabItem value="mainnet">
+  Buying tokens directly on Linea is a straightforward process, using MetaMask Portfolio's Buy feature. 
+  
+  :::info
+  ### If you already have tokens on another network, you can [follow the instructions here](./bridges-of-linea/bridging-non-dev.mdx) to bridge them to Linea.
+  :::
+
+**Linea purchases in MetaMask Portfolio's Buy feature are available worldwide.**
+
+<div class="center-container">
+    <div class="video-container">
+      <iframe
+        class="video-iframe"
+        src="//www.youtube.com/embed/BOPDw348-zg"
+        title="TITLE"
+        allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+        allowFullScreen></iframe>
+    </div>
+</div>
+
+:::info
+### What is MetaMask Portfolio?
+
+[Portfolio](https://portfolio.metamask.io/) is a dapp built by our friends at MetaMask that allows you to buy, sell, swap, stake, and manage your cryptoassets, among other features. The Buy feature is one of these functions, and is the same service available [through MetaMask](https://support.metamask.io/hc/articles/360058239311).
+:::
+
+## Steps to buy tokens on Linea
+
+1. Navigate to [portfolio.metamask.io](https://portfolio.metamask.io), and connect the account to which you want your purchased tokens to be delivered.
+   
+2. Select 'Buy' from the hamburger icon menu located in the top left of the screen.
+   
+3. Now you'll see the Buy interface. If you've never used it before, you'll first need to **select your region**. 
+
+  This information is required because availability of payment methods and providers vary based on your bank's location.
```

Comment:
I think this line (145) needs to match the indentation of step 3 that it belongs to, otherwise it breaks the formatting of the numbered list, which impacts readability/flow